pkg/network/tracer/connection/ebpfless_tracer.go

```diff
@@ -199,9 +200,15 @@ func (t *ebpfLessTracer) processConnection(
 	var err error
 	switch conn.Type {
 	case network.UDP:
+		if (ip4 != nil && !t.config.CollectUDPv4Conns) || (ip6 != nil && !t.config.CollectUDPv6Conns) {
```

Comment:
Should we also just skip if `ip4/6 == nil`? Ditto for tcp

Reply:
Done in the previous UDP PR